specs/core/0_beacon-chain.md

```diff
@@ -69,7 +68,7 @@
             - [`compute_shuffled_index`](#compute_shuffled_index)
             - [`compute_proposer_index`](#compute_proposer_index)
             - [`compute_committee`](#compute_committee)
-            - [`compute_epoch_of_slot`](#compute_epoch_of_slot)
+            - [`compute_epoch_at_slot`](#compute_epoch_at_slot)
             - [`compute_start_slot_of_epoch`](#compute_start_slot_of_epoch)
```

Comment:
should be `compute_start_slot_at_epoch` for consistency with `compute_epoch_at_slot`, `get_block_root_at_slot` and `get_committee_count_at_slot`.